c2corg_ui/static/js/apiservice.js

```diff
@@ -399,17 +399,23 @@ app.Api.prototype.updatePreferredLanguage = function(lang) {
 
 /**
  * @param {undefined | string} token
- * @param {string} lang
+ * @param {undefined | string} lang
  * @param {?number} userId
  * @param {boolean} isPersonal
  * @return {!angular.$q.Promise<!angular.$http.Response>}
  */
-app.Api.prototype.readFeed = function(token, lang, userId, isPersonal) {
+app.Api.prototype.readFeed = function(token, lang, userId, isPersonal, isChangesFeed) {
```

Comment:
Since those are really different feeds, I would recommend to use dedicated functions.